src/tools/applive-utils/start-session.ts

```diff
@@ -221,6 +221,8 @@ function openBrowser(launchUrl: string): void {
         : process.platform === "win32"
           ? ["cmd", "/c", "start", launchUrl]
           : ["xdg-open", launchUrl];
+    
+    // nosemgrep:javascript.lang.security.detect-child-process.detect-child-process
 
```

Comment:
add this just above the `const child...` code line, otherewise this will not work.